draft-ietf-rats-architecture.md

```diff
@@ -1286,6 +1286,11 @@ This approach is often called "Direct Anonymous Attestation".  See
 
 # Security Considerations {#security-considerations}
 
+This document provides an architecture for doing remote attestation.
+No specific wire protocol is documented here.
+Without a specific proposal to compare against, it is impossible to know if the the security  considerations listed below have been mitigated well.
```

Comment:
```suggestion
Without a specific proposal to compare against, it is impossible to know if the security considerations listed below have been mitigated well.
```
1) Fix double "the". 
2) Re "considerations ... mitigated":  you mitigate threats... are all security considerations threats?
3) some considerations aren't part of the implementation per se, like the discussion of physical security at a factory.  Would you consider those part of "a specific proposal"?